procedures/katello/release.md.erb

```diff
@@ -60,4 +61,5 @@
 - [ ] Post a [release announcement](https://community.theforeman.org/c/release-announcements/8)
 <% unless is_rc %>
 - [ ] Create and upload [release specific developer stable box](https://github.com/theforeman/forklift/blob/master/packer/README.md#creating-version-specific-katello-devel-boxes)
+- [ ] Remove references to the oldest Katello version from [testKatello.groovy](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/test/testKatello.groovy), [katello-pipelines.yml](https://github.com/theforeman/jenkins-jobs/blob/master/centos.org/jobs/katello-pipelines.yml), and [<%= short_version %>.groovy](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/katello/4.3.groovy).
```

Comment:
Note this process is also used for .z releases. This should only be limited to non-RC and x.y.0 releases since for x.y.1 you can assume it's already gone. It won't hurt that much, but it's good to not have redundant steps :)

Also, you missed a version in the URL
```suggestion
- [ ] Remove references to the oldest Katello version from [testKatello.groovy](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/test/testKatello.groovy), [katello-pipelines.yml](https://github.com/theforeman/jenkins-jobs/blob/master/centos.org/jobs/katello-pipelines.yml), and [<%= short_version %>.groovy](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/katello/<%= short_version %>.groovy).
```

Reply:
I purposefully left that version out because the file needs to be created, so the link will always be a 404 for first-time releases.  I can make it more clear that the link is to an example.

Reply:
Actually, I can take out the reference to x.y.groovy because nothing should be deleted from it anyway